app/views/users/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>All Users</h1>
-<!-- Placeholder for listing all users -->
+<main id="home" class="home-page">
+  <ul class="users-list">
+    <% User.all.each do |user| %>
+      <li class="user-container">
+        <div class="profile-pic">
+          <%= link_to user_path(user) do %>
+            <img src="<%= user.photo %>" alt="profile-picture">
+          <% end %>
+        </div>
+        <div class="class-info">
+          <h3><%= link_to user.name, user_path(user)  %></h3>
+          <p class="posts-counter">Number of posts: <%= user.posts_counter || 0 %></p>
+        </div>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
- After adding some data to your database, the users are not been updated and no users seem to show in your UI. Please, kindly fix this.

![image](https://github.com/WilhelmK109/BlogApp/assets/59397159/b0b45126-af33-4235-9ae2-0b7221c9c4c7)